modules/TorqueSwerveModule2021.java

```diff
@@ -195,6 +196,10 @@ public final SwerveModuleState getState() {
         return new SwerveModuleState(encoderPerMinuteToMetersPerSecond(drive.getVelocityRPM()), getRotation());
     }
 
+    public SwerveModulePosition getPosition() {
+        return new SwerveModulePosition(drive.getPosition(), getRotation());
```

Comment:
drive.getPosition has to return meters that the wheel would go. This would return RPMs that the motor drove or something. You have to do the math. 

Then again, I don't think that this is necessary to add because there isn't a robot using these modules that needs auto and odometry.

Reply:
I see, I'll try to resolve this today